src/app/main-page/main-page/main-page.component.ts

```diff
@@ -1,15 +1,14 @@
-import { Component, OnInit } from '@angular/core';
-
-@Component({
-  selector: 'app-main-page',
-  templateUrl: './main-page.component.html',
-  styleUrls: ['./main-page.component.scss']
-})
-export class MainPageComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
-}
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-main-page',
+  templateUrl: './main-page.component.html',
+  styleUrls: ['./main-page.component.scss']
+})
+export class MainPageComponent {
+
+  public onLoadCourse(): void {

```

Comment:
It would be better to mention event type in the event handler name. It makes code easier to read and maintain. I suggest renaming it to `onLoadCourseClick`